ts/storage.ts

```diff
@@ -0,0 +1,20 @@
+import { TypeTodo } from "./demoTodo.js";
+
+const storage = {
+  get: function (key: string) {
+    const keyItem = localStorage.getItem(key);
+    if (keyItem) {
+      let value: string = JSON.parse(keyItem);
```

Comment:
Why is it always string ? What if `JSON.parse` results to array, object, number ?